docs/src/lib/indexing.md

```diff
@@ -223,3 +221,12 @@ The elements of a `GroupedDataFrame` are [`SubDataFrame`](@ref)s of its parent.
 * `gd[key::GroupKey]` -> Get the group corresponding to the [`GroupKey`](@ref)
   `key` (one of the elements of the vector returned by [`keys(::GroupedDataFrame)`](@ref)).
   This should be nearly as fast as integer indexing.
+* `gd[a::AbstractArray]` -> Select multiple groups and return them in a new
+  `GroupedDataFrame` object. Groups may be selected by integer position using an
+  array of `Integer`s or `Bool`s, similar to a standard array. Alternatively the
+  array may contain keys of any of the types supported for dictionary-like
+  indexing (`GroupKey`, `Tuple`, or `NamedTuple`). Selected groups must be
+  unique.
+* `gd[n::InvertedIndex]` -> Any of the above types wrapped in
```

Comment:
maybe use `Not` here and everywhere in the code instead of `InvertedIndex` as this is what users use (so it will be easier to read the code in case they need to).

Reply:
Is `Not` supported with a `@ref` link?

Reply:
No, but neither `InvertedIndex` is - it is a limitation of Documenter.jl AFAICT - you cannot `@ref` to another package.